core/primitives/src/trie_key.rs

```diff
@@ -74,7 +74,7 @@ pub mod col {
         (PROMISE_YIELD_RECEIPT, "PromiseYieldReceipt"),
     ];
 
-    pub const ALL_COLUMNS_WITH_NAMES: [(u8, &'static str); 15] = [
+    pub const ALL_COLUMNS_WITH_NAMES: [(u8, &'static str); 16] = [
```

Comment:
This object is accessed in a few places to translate a col index into a col name, to display errors.
There was a hole at position 8 that I'm filling with an `unused` value.